packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -45,25 +46,53 @@ def draw(self):
             self.figure.dpi = orig_dpi
             self._idle_scheduled = False
 
+    def print_png(self, filename_or_obj, *args,
+                  metadata=None, pil_kwargs=None,
```

Comment:
I think `pil_kwargs` is unnecessary here.

Reply:
I took inspiration from the `agg` backend here: https://github.com/matplotlib/matplotlib/blob/db2193ccadeaf022ff0fc4706a9c03db471a8e06/lib/matplotlib/backends/backend_agg.py#L459

Let me know if we still want to remove it?

Reply:
Yes, please remove it.  It's not used in the function, and if it were passed in it would be put into the `**kwargs` below.

Reply:
Sure, on it!